### DISCUSSION TITLE
拍一張照片就能讓AI幫你設計房間？

### DISCUSSION THREAD

**Discussion by Miinteaa** [1101-GS4719-C/Python-Community-GS4719-C#1473] [General]
最近看到一個線上AI機器人叫做「**RoomGPT**」，其功能為透過使用者**上傳室內照片並選擇裝潢風格，就能在幾秒內自動生成整體裝潢**。

這個功能對於想要改造或統一房間擺設風格的人們來說，是非常便利的工具；對於室內設計師來說，也能夠透過RoomGPT在與顧客討論的當下產生示意圖，省去等待設計師建模繪圖的時間，馬上看到效果並與客人溝通，大大提升了工作效率。

但對於裝潢細節微調或者是材料的評估等等，還是有所限制，畢竟要做出實際和AI產生圖例100%相同的裝潢也是難上加難。
另外舉例如施工圖等等還是需要人工繪製及調整，這些目前也無法單純依靠AI產生。
且因AI主要是利用使用者上傳的照片套用模板風格，無法直接用空房間產生一個完整的房間布置，且不會改變家具擺設原本位置（ex: 沙發位置相同，只改變顏色或配件、材質、外觀設計等等），缺少創意發想，這些部分依然需要透過室內設計師或人們自己實現。

不過如果只是單純想要看看不同風格在自己房間格局底下的呈現，使用RoomGPT是個不錯的模擬方法！
在此附上RoomGPT網址：https://www.roomgpt.io/

**Comment by alstinlin** [comment]
聽起來很不錯! 雖然限制還很多，但已經幫設計師和顧客減少許多時間，而且roomgpt如果在創意方面也能提供不錯的想法或概念，未來一定可以在居家裝潢上有更多的應用

**Comment by Aaron0810** [comment]
講到房間設計就讓我想到，前陣子甚至到最近都還有，我的fb有時候會跳出來的中國房間設計抖音，他會把房間設計的非常理想，講的好像很厲害又有做動畫出來模擬，但在youtube上就會有一些設計師頻道把這些小短片拿出來講，一個一個拿出來吐槽，雖然不知道這個RoomGPT實際上如何，但感覺肯定比中國幹片好用

**Comment by juliaouo** [comment]
當面對一個空白的房間，不確定該怎麼開始，RoomGPT就能作為一個很棒的靈感來源。能夠展示不同的裝潢風格，讓我們在沒有設計師的情況下，對於不同風格的擺設有更清晰的想像。這對於那些對於自己家的風格尚未確定的人來說，真的是一個很好的模擬方法。希望後續它能帶來更多驚喜和創意並提供更多可行性！

**Comment by ashleyy-huang** [comment]
這項科技非常的實用，不論是買新房子，又或是想重新裝潢，這項科技帶給我們非常大的幫助，但這顆季的到來也意味著設計師的工作岌岌可危，這項科技雖說現在還沒版法完全取道設計師，但隨著現今科技的演進，有朝一日一定能夠完全被取代的，所以說現在的我們應該想想如何進修自己，才不會被人工智慧給取代。

**Comment by lyien1021** [comment]
我有試用了一下，怎麼感覺只是一個噱頭啊🤣不過不可否認如果能繼續精進這個AI模型，的確會對設計有很大的幫助～

**Comment by Superlan8** [comment]
我覺得它可以給一開始完全沒方向要怎麼設計的人一些想法，但我認為它目前技術還沒到可以取代設計師的工作，比如一些客製化的東西，個人生活使用習慣，以及房間採光等，這些還是目前AI沒辦法做到的事。

**Comment by Brownnnn78** [comment]
我覺得室內設計師不可替代的是在設計時不只要考慮風格，還需要考量各個細節，例如插座要放哪、放幾個，傢俱的長寬高適合的比例和現有能夠買到的家俱中cp值最高的，或是家裡的隔間要不要請額外請木工之類的各式各樣的該注意的，並不是畫畫圖就結束了！

**Comment by youtinglin** [comment]
好棒的功能，讓大家可以有些創造靈感在空間設計上，若這項技術能在之後結合真實的木工材料和實用性的考量，加上對於一些工程上的技術評估(材質的耐震及裝置上的安全性及可行性)，不僅能在美觀上有作用，並能更貼近真實使功能更完善。

**Comment by Tzuhsuannnnn** [comment]
好酷的新功能! 而且設計通常是人們比較不熟悉的領域，請設計師也是一筆金額，這個功能可以讓AI直接幫我們設計，超方便的!
